src/igvfd/report.py

```diff
@@ -56,20 +62,37 @@ def _convert_camel_to_snake(type_str):
 @view_config(route_name='report_download', request_method='GET')
 def report_download(context, request):
```

Comment:
Leave this endpoint alone and make new endpoint e.g. `multitype_report_download`.